### PR TITLE
Small corrections to Delphes FCCSW interface

### PR DESCRIPTION
**Pull request by selvaggi**

### CODE REVIEW

**Comment by jlingema** [comment]
Hi Michele,
This also needs the new Delphes version, right? If so, I'll install in the pre-release area so we can test the PR.
Cheers,
Joschka

**Comment by selvaggi** [comment]
Yes, that would be great, thanks.

**Comment by selvaggi** [comment]
Added new features such as event weight and and gen particle skimming.
Skimming originally planned to be donned in Delphes, but it gives more flexibility to do it FCCSW.
The un-skimmed MC particle collection is not stored by default, but can be easily done by removing the relevant drop in the PythiaDelphes_config.py file.

**Comment by jlingema** [comment]
OK, this now also depends on HEP-FCC/fcc-edm#30

**Comment by jlingema** [comment]
ok to test

File: Sim/SimDelphesInterface/options/PythiaDelphes_config.py [hunk 1]
```diff
@@ -38,29 +38,29 @@ def apply_paths(obj, names):
...
+  
 ############################################################
 #
 # User: Configure variables
 #
 ############################################################
 
 ## N-events
-nEvents=100
+nEvents=1000
```
Comment (jlingema) [inline Sim/SimDelphesInterface/options/PythiaDelphes_config.py]: Can we change this back to `100` to make the tests for the PRs shorter?

**Comment by selvaggi** [comment]
done

**Comment by jlingema** [comment]
retest this please

**Pull request merged**